### PR TITLE
Adding jfrog_metrics integration tile

### PR DESCRIPTION
**Pull request by MahithaB**

### CODE REVIEW

File: jfrog_metrics/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - jfrog_metrics
+All changes to the JFrog metrics integration will be documented in this file.
+
+## [0.1.0] - Apr 16, 2021
```
Comment (MahithaB) [inline jfrog_metrics/CHANGELOG.md]: can you please change this date

File: jfrog_metrics/datadog_checks/jfrog_metrics/__about__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```
Comment (MahithaB) [inline jfrog_metrics/datadog_checks/jfrog_metrics/__about__.py]: version in changelog and __about.py__ should be the same. Can you please change accordingly

Comment (MitaliBo) [reply jfrog_metrics/datadog_checks/jfrog_metrics/__about__.py]: why do we have version 0.1.0 in changelog? should not it be 0.0.1

Comment (MahithaB) [reply jfrog_metrics/datadog_checks/jfrog_metrics/__about__.py]: Since it is the first major version, and according to our versioning format, 1.0.0

**Review by vinayagg — APPROVED** [review]

**Pull request merged**